src/components/Proposal/View/Categories/GrantProposalView.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+
+import Paragraph from 'decentraland-gatsby/dist/components/Text/Paragraph'
+import useFormatMessage, { useIntl } from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+
+import { NewGrantCategory } from '../../../../entities/Grant/types'
+import { GrantProposalConfiguration } from '../../../../entities/Proposal/types'
+import BudgetBreakdownView from '../../../GrantRequest/BudgetBreakdownView'
+import PersonnelView from '../../../GrantRequest/PersonnelView'
+import CategoryAssessment from '../../CategoryAssessment'
+import ProposalDescriptionItem from '../ProposalDescriptionItem'
+
+interface Props {
+  config: GrantProposalConfiguration
+}
+
+const CURRENCY_FORMAT_OPTIONS = {
+  style: 'currency',
+  currency: 'USD',
+  maximumFractionDigits: 0,
+}
+
+function GrantProposalView({ config }: Props) {
```

Comment:
There's a `ProposalView.css` that is empty and not being imported here, I think you can remove it :)